docs/source/metrics.rst

```diff
@@ -37,6 +37,13 @@ use the `output_transform` argument to transform it:
         metric = Accuracy(output_transform=output_transform)
         metric.attach(engine, "accuracy")
 
+Metrics could be combined together to form a new metric through arithmetics,
+for example:
+    .. code-block:: python
+        precision = Precision()
+        recall = Recall()
+        F1 = precision * recall * 2 / (precision + recall)
```

Comment:
According to the integration test of #323 this computation is not a real F1. Maybe we need replace this example by something else.

Reply:
@vfdev-5 We can just add `average=False` and then this will be the F1 for each class. I just added an integration test, you can take a look.

Reply:
@zasdfgbnm looks good !